app/printTemplates/summary/local/pdf.css

```diff
@@ -1,23 +1,143 @@
-#header { position: fixed; left: 0px; top: -25px; right: 0px; height: 55px;  text-align: left; vertical-align: top; margin: 0px 5px 10px 0px; padding: 0px 5px 2px 0px; }
-#footer { position: fixed; left: 0px; bottom: -50px; right: 0px; height: 20px; text-align: center; vertical-align: bottom; margin: 0px 5px 10px 5px; padding: 3px 5px 2px 10px; }
-#header .pagingText:after { content: counter(page, decimal); }
-#body { z-index: 30000; position: relative; }
+#header {
```

Comment:
Replaced with the version from app/printTemplates/summary/pdf.css and reformatted to make legible